.github/workflows/build.yml

```diff
@@ -1,19 +1,23 @@
 name: Deploy spec
 
-on: [push]
+on:
+  push:
+    branches:
+      - master
 
 jobs:
   build:
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-node@v1
-      with:
-        node-version: '12.x'
-    - run: npm install
-    - run: npm run build
-    - name: commit changes
-      uses: elstudio/actions-js-build/commit@v3
-      with:
-        commitMessage: "fixup: [spec] `npm run build`"
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js (LTS)
```

Comment:
we're running on node 12, not necessarily on "LTS" (which is now node 14 anyways)